src/common/interfaces.ts

```diff
@@ -1,12 +1,17 @@
-import { ReadStream } from 'fs';
+import { ReadStream, WriteStream } from 'fs';
```

Comment:
Use node:fs instead of fs
<img width="430" height="227" alt="image" src="https://github.com/user-attachments/assets/2fbc7393-1024-4a53-a1b0-c59647dc33c8" />
Check if you can use node:fs/promises as it's async api